### ISSUE TITLE
Profiles database website

### ISSUE THREAD

**Issue by AriaMoradi** [AntiMicroX/antimicrox#101]
We have a repo for profiles but it's hard for non-technical users to work with the repo and repo tree is not SEO friendly at all.
we could create a website to handle the profiles database.

It might be possible to code the site in vue and automatically store all profiles in a repo (It will be closer to a JAM stack site).

**Comment by AriaMoradi** [comment]
This issue is related to the profiles repo and should be transferred over later.

**Comment by gombosg** [comment]
...or extend the current site with a profiles menu item? :)

**Comment by AriaMoradi** [comment]
> ...or extend the current site with a profiles menu item? :)

Yep, that can be done, but we need to make metadata files for each profile and handling submissions will be a hassle.
